apps/nextjs-app-router/src/app/[lang]/[[...path]]/page.tsx

```diff
@@ -3,19 +3,27 @@ import '@/makeswift/components'
 import { getSiteVersion } from '@makeswift/runtime/next/server'
 import { notFound } from 'next/navigation'
 import { Page as MakeswiftPage } from '@makeswift/runtime/next'
-import { locales } from '@/localization'
 
 type ParsedUrlQuery = { lang: string; path?: string[] }
 
 export async function generateStaticParams() {
-  const pages = await client.getPages()
 
-  return pages.flatMap((page) =>
-    locales.map((locale) => ({
-      path: page.path.split('/').filter((segment) => segment !== ''),
-      lang: locale,
+  const pages = await client.getPages().toArray()
+
+  return pages.flatMap((page) => [
+    {
+      params: {
+        path: page.path.split('/').filter((segment) => segment !== ''),
```

Comment:
What is `page.path.split('/').filter((segment) => segment !== '')` for?

Reply:
I assume this is to avoid empty path segments in the parsed outputs, which is important given the leading slash. When you do `"/test".split("/")`, for example, the first member of the array is an empty string. Similarly, doing `"/".split("/")` results in `["", ""]`